birdhouse/optional-components/weaver/default.env

```diff
@@ -0,0 +1,46 @@
+# ------
+# weaver
+# ------
+
+EXTRA_VARS='
+  $WEAVER_CONFIG
+  $WEAVER_MANAGER_NAME
+  $WEAVER_WORKER_NAME
+  $WEAVER_WPS_NAME
+  $WEAVER_WPS_OUTPUTS_DIR
+  $WEAVER_WPS_OUTPUTS_PATH
+  $WEAVER_WPS_PATH
+  $WEAVER_WPS_WORKDIR
+'
+
+# extend the original 'VARS' from 'birdhouse/pavics-compose.sh' to employ them for template substitution
+# adding them to 'VARS', they will also be validated in case of override of 'default.env' using 'env.local'
+VARS="$VARS $EXTRA_VARS"
+
+# configuration can be EMS|ADES
+# the corresponding "/<config>" route will redirect to the protected 'weaver' service
+export WEAVER_CONFIG=ADES
+# endpoint where Weaver will be accessible from
+export WEAVER_VERSION=1.14.0
+# real names of the weaver/worker services
+# these values will generate "<server-fqdn>/<name>" URI to access them behind secured proxy
+export WEAVER_MANAGER_NAME=weaver
+export WEAVER_WORKER_NAME=worker
+export WEAVER_WPS_NAME=${WEAVER_MANAGER_NAME}-wps
+# WPS-1/2 path under Weaver's root endpoint
+export WEAVER_WPS_PATH=/wps
+# data persistence of wps outputs (directory path)
+# for the moment, this is hardcoded in 'birdhouse/docker-compose.yml'
+export WEAVER_WPS_OUTPUTS_DIR="${DATA_PERSIST_ROOT}/wps_outputs"
```

Comment:
Following my previous comment on `WEAVER_WPS_OUTPUTS_DIR` : So you choose to dump result under the same directory but allow it to be different. Other birds dump their results into wps_outputs/<bird_name>. To respect this convention, should we set weaver output dir to `${DATA_PERSIST_ROOT}/wps_outputs/weaver` ?

Reply:
Yes. We can change it to follow this same convention.